firmware/controllers/algo/advance_map.cpp

```diff
@@ -169,6 +176,8 @@ void setDefaultIatTimingCorrection(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 void prepareTimingMap(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	advanceMap.init(config->ignitionTable, config->ignitionLoadBins,
 			config->ignitionRpmBins);
+	advanceTpsMap.init(CONFIG(ignitionTpsTable), CONFIG(ignitionTpsBins),
+			config->ignitionRpmBins);
```

Comment:
We init both tables in RAM because here we're at a very early stage, with no config settings loaded.